ManiVault/src/actions/OptionAction.cpp

```diff
@@ -388,8 +388,8 @@ void OptionAction::ComboBoxWidget::paintEvent(QPaintEvent* paintEvent)
 
     initStyleOption(&styleOptionComboBox);
 
-    if (_optionAction->getCurrentIndex() == -1)
-        styleOptionComboBox.currentIcon = Application::getIconFont("FontAwesome").getIcon("list");
+    //if (_optionAction->getCurrentIndex() == -1)
+    //    styleOptionComboBox.currentIcon = Application::getIconFont("FontAwesome").getIcon("list");
```

Comment:
Doing a [quick search](https://github.com/search?q=org%3AManiVaultStudio+setCurrentindex%28-1&type=code) `setCurrentindex(-1)` is actually used in the OptionAction itself and the ImageLoader. I'm not sure if this has any consequences?

Reply:
No, this has only cosmetic influence (Iicon)